docs/topics/api/addons.rst

```diff
@@ -166,6 +166,7 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
     :>json boolean has_privacy_policy: The add-on has a Privacy Policy (See :ref:`add-on EULA and privacy policy <addon-eula-policy>`).
     :>json string|object|null homepage: The add-on homepage (See :ref:`translated fields <api-overview-translations>`).
     :>json string icon_url: The URL to icon for the add-on (including a cachebusting query string).
```

Comment:
If we're keeping the old `icon_url` around can we slightly alter the description to mention it's the default 64x64 size icon it returns.